base/setup/usetup/lang/es-ES.h

```diff
@@ -1534,14 +1534,14 @@ static MUI_ENTRY etESBootLoaderInstallPageEntries[] =
     {
         4,
         3,
-        " ReactOS " KERNEL_VERSION_STR " Setup ",
+        " Instalaci\242n de ReactOS " KERNEL_VERSION_STR " ",
         TEXT_STYLE_UNDERLINE,
         TEXT_ID_STATIC
     },
     {
         0,
         0,
-        "Installing the bootloader onto the media, please wait...",
+        "Instalando sector de arranque en el medio, espere un poco...",
```

Comment:
I would preffer another word rather than "en el medio", because could be understood as "in the middle", alternative words could be "en el dispositivo" or "en el disco"

Reply:
Also, sounds more respectful "por favor espere" instead of "espere un poco"

Reply:
1. The first one, I will take your suggestion, but changing from "Medio" to "Disco". It's not the real translation, but it might be better understood in the ReactOS context, that noone used the "medio" term (like medio de almacenamiento, storage media). But let me say that It's not incorrect (and Windows XP used it a lot). Also could be used "Unidad". "Dispositivo" I think never because is too generic.
2. Ok, accepted. Let's be more respectful.
```suggestion
        "Instalando sector de arranque en el disco, espere por favor...",
```